index.js

```diff
@@ -0,0 +1,201 @@
+//Task №1
+
+const videos = [
+	{
+	id: 65432445,
+	title: 'The Chamber',
+	},
+	{
+		id: 675465,
+		title: 'Fracture',
+	},
+	{
+		id: 70111470,
+		title: 'Bad Boys',
+	},
+	{
+		id: 654356453,
+		title: 'Die Hard',
+	},
+];
+
+const videoTitleById = (videoItem) => videoItem.reduce((result, video) => {
```

Comment:
your function apply the list of videos, not the `videoItem`. please, rename the arg